.github/workflows/verify-dependabot.yml

```diff
@@ -0,0 +1,50 @@
+name: Verify (dependabot)
+
+on:
+  pull_request_target:
+    branches:
+      - main
+
+env:
+  NODE_VERSION: 14.x
+
+jobs:
+  build:
+    name: Build and Test
+    runs-on: ubuntu-latest
+    if: github.actor == 'dependabot[bot]'
+    steps:
+      - name: Cancel Previous Runs
+        uses: styfle/cancel-workflow-action@0.9.1
```

Comment:
Do we need this or can we get away with using Github Actions concurrency feature?

Reply:
Clearly don't need this, we should go for Github Actions concurrency